howto/inference.md

```diff
@@ -53,3 +53,26 @@ python generate.py --quantize gptq.int4
 
 For instance, `falcon-40b` "only" requires ~24 GB to generate using this technique, otherwise more than +40GB are required. On an A100 it takes 1 token/sec.
 However, ~32 GB were required during the conversion step.
+
+
+## Run a large model on multiple consumer devices
+
+You can also use the Fully-Sharded Data Parallel (FSDP) distributed strategy to leverage multiple devices to perform inference. This will allow you to run models that wouldn't fit in a single card by sharding them across several.
+
+For instance, `falcon-40b` would require ~80 GB of GPU memory to run on a single device. We can instead run it on 4 A100 40GB GPUs:
+
+```shell
+python generate/base.py --checkpoint_dir checkpoints/tiiuae/falcon-40b --strategy fsdp --devices 4
+```
+
+Which will take 32 GB of memory, and run at 0.37 tokens/sec.
+
+Or to reduce the memory requirements even further, you can try using CPU offloading. For that, you will need to manually edit the `cpu_offload=False` parameter in the file and set it to `True`.
+
+Now we can run it on just 2 A100 40GB GPUs:
```

Comment:
If it takes 13GB of memory, then it wouldn't need 2 A100s. We could say 3090s or similar consumer hardware